### ISSUE TITLE
Brew issues

### ISSUE THREAD

**Issue by bmicklea** [jozu-ai/kitops#563]
**Describe the bug**
There are some issues with the `kitops.rb` file in our brew install:
* Points to `brett-hodges` organization for the download (should pull from main kitops)
* File a PR for a new cask: https://github.com/Homebrew/homebrew-cask?tab=readme-ov-file#requests

**Comment by brett-hodges** [comment]
@bmicklea Regarding the first issue:  The kitops.rb file is automatically generated by the KitOps build process.  Since we (Jozu)  haven't built KitOps since before I added the brew install functionality, the kitops.rb file hasn't been updated.  When we (Jozu) do build the next version of KitOps, the urls in the kitops.rb file should point to jozu-ai and not brett-hodges.

**Comment by bmicklea** [comment]
@gorkem can we cut a KitOps release today? We've moved Brew into our KitOps docs so I'd like to make sure everything works as expected (it worked for me).

**Comment by bmicklea** [comment]
So a new Kit release was cut last week (https://github.com/jozu-ai/kitops/releases/tag/v0.3.4), but the `.rb` file hasn't changed: https://github.com/jozu-ai/kitops/tree/main/build/homebrew

Are you sure it auto-updates when a new Kit version is built? Did a step fail or skip last week?

**Comment by brett-hodges** [comment]
There was an issue with the way I was setting an environment variable in the Github Action that builds the .rb file.  I made the change last week.  @amisevsk Can we push a new release of Kitops to enable the Homebrew installation?

**Comment by amisevsk** [comment]
I believe we're waiting on a couple of PRs to be merged before cutting a new minor release

**Comment by bmicklea** [comment]
Can you add links to those so we can stay on top of getting the release cut?

**Comment by bmicklea** [comment]
Nice - that will be an impactful release:
1. New Kit CLI config command and `env` file
1. `kit dev` support for Windows, Linux, and MacOS
1. Brew install option for Kit CLI
1. (Stretch) Kit Python libraries (#118)

I wouldn't hold the release for (4) but it would make a nice addition.

**Comment by brett-hodges** [comment]
Closing this issue as the Homebrew bug is fixed as of KitOps v.0.4.0 and a separate Issue has been opened related to creating a Homebrew Cask (See the Issue Description above.)
